specification/metrics/semantic_conventions/runtime-environment-metrics.md

```diff
@@ -398,5 +402,83 @@ This metric is obtained from [`BufferPoolMXBean#getCount()`](https://docs.oracle
 **[1]:** Pool names are generally obtained via [BufferPoolMXBean#getName()](https://docs.oracle.com/en/java/javase/11/docs/api/java.management/java/lang/management/BufferPoolMXBean.html#getName()).
 <!-- endsemconv -->
 
+### Metric: `process.runtime.jvm.cpu.monitor.duration`
+
+This metric is [recommended][MetricRecommended]. Only available with JDK 17+.
+This metric is obtained from [`jdk.JavaMonitorWait`](https://sap.github.io/SapMachine/jfrevents/21.html#javamonitorwait) and [`jdk.JavaMonitorEnter`](https://sap.github.io/SapMachine/jfrevents/21.html#javamonitorenter) JFR events.
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument-advice)
```

Comment:
Why are we recommending empty buckets?  Do we not care about percentiles here?

Reply:
I agree it would be better to have a set of buckets, but I'm not sure what to best recommend. A thread could be blocked/waiting for any length of time. There is also a [single bucket precedent for GC pause time](https://github.com/open-telemetry/semantic-conventions/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md?plain=1#L195). Maybe exponential buckets would make sense.

@jack-berg what was your reasoning for [suggesting](https://github.com/open-telemetry/semantic-conventions/pull/44#discussion_r1199304308) a single bucket:
>  I recommend we downgrade this to a summary by default by specifying an empty set of explicit bucket bounds using advice.

Reply:
Reasoning is as follows:
- We need a recommended set of default buckets since the SDK defaults won't be useful.
- Coming up with sensible defaults is hard, requiring real world data for some distribution of apps.
- A single bucket histogram, or effectively a summary, probably gives most people (> 50%) the data they want. E.g. you can look at the max time a monitor is held, see how much time monitors are waiting as a rate. The distribution of the monitor wait times is a nice to have and more niche. Let those people who need it opt into it with views, but don't burden everybody by default with the extra data.

Reply:
I don't buy the reasonable defaults argument in all cases. I agree a summary is useful, but I still think there's likely SOME baseline buckets that could be provided.